README.md

```diff
@@ -261,17 +271,17 @@ sourceFile2.py.zip
 sourceFolder2.zip
 ```
 
-*NB:* When zipping a folder, the root of the archive will contain the contents
-of that folder, not the folder by itself.
+Both the `hash` and `zip` flag can be used together. This will yield the same
+result as having just the `zip` flag, except all file paths will be
+prefixed with the hash of their contents.
+
+*NB:* When zipping a folder, the structure of the folder is flattened since AWS
+Lambda can only detect source files in the root of an archive.
```

Comment:
I'm not sure this is particularly desirable. If people need to place their lambda entry points at the root of the archive, then they should just do that, rather than rely upon folders being stripped. I don't see any reason to prevent the use of folders in all cases to support this. This could be problematic for non-trivial lambdas that use a folder structure to contain source files.